azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/CallingService.kt

```diff
@@ -188,16 +184,13 @@ internal class CallingService(
             }
         }
 
-        /* <RTT_POC>
-        coroutineScope.launch {
-            callingSdk.getRttSharedFlow().collect {
-                rttStateSharedFlow.emit(it)
-            }
-        }
-        </RTT_POC> */
         return callingSdk.startCall(cameraState, audioState)
     }
 
+    fun sendRttMessage(message: String, isFinalized: Boolean) {
+        callingSdk.sendRttMessage(message, isFinalized)
```

Comment:
you may want to trim here, as sendMessage can fail if string is too long

Reply:
It is trimmed on the UI layer. I think we should not quietly hide the fact that callingSdk.sendRttMessage("too long message", isFinalized) is actually sends a shorter message.